.github/actions/matches/action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
   pattern:
     description: A grep match pattern used to find changes, must excape quotes (") if they occur in the file name...  why would you do that?.
     required: true
+  invert:
```

Comment:
Figured I'd do rust build whenever anything that's not a document/docker only change.